webpack/envs/clientDevelopmentConfig.js

```diff
@@ -76,29 +69,18 @@ export const clientDevelopmentConfig = {
   },
   name: "force",
   plugins: [
-    new CaseSensitivePathsPlugin(),
     new webpack.HotModuleReplacementPlugin(),
-    new SimpleProgressWebpackPlugin({
```

Comment:
We can use [this v5 supported version](https://github.com/clessg/progress-bar-webpack-plugin/blob/master/package.json#L30)

Reply:
This hides errors. I'm willing to add it behind a flag.

Reply:
A flag sounds good, but only if its set to on by default to preserve existing status quo behavior 👍

Reply:
It also prevents us from easily recording stats, so I would prefer not to. It outputs its noise to stdout with pollutes the stats.json file.

Reply:
I'm sorry, we're not going to upset the status quo here. It's worked fine for years without any problems until suddenly you claim it's broken. If there's a bug, provide evidence. If there's a flag, preserve the true default and you can set it false on in your .env.

Reply:
The status quo is staying on Webpack 4, with Webpack 5 there are a lot of options added and removed. As well, this is an appropriate time to investigate removing the old plugins and trying out the new functionality. Webpack provides a lot of very useful output that simple progress plugin hides and that users of Webpack would benefit from having a familiarity with. I understand some people don't like seeing logs, however, they exist for a reason.

Additionally, this plugin is nearly unused from an npm stats perspective making it a burden for us to maintain if something goes wrong and additionally the repo has been inactive for years. The link you provided is to one of their dependencies and "v5" support was added prior to Webpack 5 being released which doesn't add any comfort.

Very specifically the "Webpack is starting" line causes issues with stats files and at this point. At this point, I do think we are confounding "the status quo" with "personal preferences". 

This is also a good time to investigate adding any new plugins that are actively supported to the pipeline.

FWIW, Novo pages don't support any of these plugins yet. Which is not a call to make PRs adding them.

Reply:
To be absolutely clear: The status quo is a developer types `yarn start` to boot force. There's a sometimes minutes-long process of compiling the code, after which they can go to their browser and start interacting with the app. During those long minutes of waiting, there's a status indicator so that they know something clearly is happening. Currently, thats this progress bar. It could be any progress bar / indicator. 

> Additionally, this plugin is nearly unused from an npm stats perspective

This is [not true](https://www.npmjs.com/package/progress-bar-webpack-plugin).  

Examples: 

#### Master

![4](https://user-images.githubusercontent.com/236943/110715623-77dffc00-81ba-11eb-9b7e-431263113bc1.gif)

#### This branch 

![5](https://user-images.githubusercontent.com/236943/110715640-80d0cd80-81ba-11eb-8190-b7a949d617e6.gif)

In the v5 version the developer is just sitting there looking at silence, with a message saying "Listening on localhost:5000" 

We don't need to use the plugin linked above (although with ~112k weekly downloads its surely fine), but we need to preserve the status quo, which is feedback for our developers during compilation and boot. This has nothing to do with personal preference.

Reply:
> Webpack provides a lot of very useful output that simple progress plugin hides and that users of Webpack would benefit from having a familiarity with

The progress bar linked above, which includes expected default log output on complete: 

![5-progress](https://user-images.githubusercontent.com/236943/110718701-5f72e000-81c0-11eb-96c7-4b7e2e283251.gif)